static/app/views/performance/browser/webVitals/overallProgressRing.tsx

```diff
@@ -0,0 +1,198 @@
+import {Fragment, useRef, useState} from 'react';
+import {css, useTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import useMouseTracking from 'sentry/utils/replays/hooks/useMouseTracking';
+import PerformanceScoreRing from 'sentry/views/performance/browser/webVitals/components/performanceScoreRing';
+import {
+  PERFORMANCE_SCORE_WEIGHTS,
+  ProjectScore,
+} from 'sentry/views/performance/browser/webVitals/utils/calculatePerformanceScore';
+import {WebVitals} from 'sentry/views/performance/browser/webVitals/utils/types';
+
+import {ORDER} from './performanceScoreChart';
+
+const {
+  lcp: LCP_WEIGHT,
+  fcp: FCP_WEIGHT,
+  fid: FID_WEIGHT,
+  cls: CLS_WEIGHT,
+  ttfb: TTFB_WEIGHT,
+} = PERFORMANCE_SCORE_WEIGHTS;
+
+type Props = {
+  height: number;
+  projectScore: ProjectScore;
+  ringBackgroundColors: string[];
+  ringSegmentColors: string[];
+  text: React.ReactNode;
+  width: number;
+  hideWebVitalLabels?: boolean;
+};
+
+function OverallProgressRing({
```

Comment:
nit: Maybe we should name this something like `PerformanceScoreRingWithTooltips`. And we can move this to the components folder too.